source/main/gameplay/RoRFrameListener.h

```diff
@@ -25,6 +25,8 @@
 
 #include <Ogre.h>
 
+namespace RoR { struct Terrn2Telepoint; } // Forward decl.

```

Comment:
Nope. We have namespaces `RoR`, `RoR::App`, `RoR::GUI` and that is not going to change.


@AnotherFoxGuy Please remove this rule or update the regex.